starter-code/main.js

```diff
@@ -0,0 +1,117 @@
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const url = `mongodb://localhost:27017/crunchbase`
+
+mongoClient.connect(url, (error, db) => {
+  if (error) {
+    console.log('Error trying to connect to the Database');
+    console.log(error);
+  } else {
+    console.log('Connection established correctly!! 😬');
+
+    function mainMenu() {
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) => {
+        switch (option) {
+          case "1":
+            db.collection('companies').find({}, {
+              name: 1,
+              _id: 0
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu()
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu()
+                });
+              }
+            })
+            break;
+          case "2":
+            db.collection('companies').count({}, (error, result) => {
+              console.log("Nb of companies:", result);
+              rl.question(`\nType enter to continue: `, (answer) => {
+                mainMenu()
+              });
+            });
+
+            break;
+          case "3":
+          db.collection('companies').count({"founded_year" : '2004'}, (error, result) => {
+
+            if (error) {
+              console.log(error);
+              rl.question(`\nType enter to continue: `, (answer) => {
+                mainMenu()
+              });
+            } else {
+              console.log(result);
+              rl.question(`\nType enter to continue: `, (answer) => {
+                mainMenu()
+              });
+            }
+          });
+            break;
+            case "4":
+            db.collection('companies').find({ $and: [{"founded_year" : 2004},{"february" : 2}] (error, result) => {
```

Comment:
You are using `february` although `2` already means `february`. What do you want to say is that the month must be equal to 2. As the month is stored in the `founded_month` variable, you need to write `{ founded_month: 2 }`.

Also, you need to close the object before passing the callback.

In the end, the line will look like that:

```javascript
db.collection('companies').find({ $and: [{"founded_year" : 2004}, {"founded_month" : 2}] }, (error, result) => {
```